src/translations/sk.json

```diff
@@ -0,0 +1,41 @@
+{
+    "log_in": "Prihlásiť sa",
+    "log_out": "Odhlásiť sa",
+    "loged_in_as": "Prihlásený ako",
+    "logged_in": "Prihlásený",
+    "logging_in": "Prihlásenie",
+    "logging_out": "Odhlásenie",
+    "sign_up": "Zaregistrovať sa",
+    "signing_up": "Registrácia",
+    "remember_password": "Zabudli ste heslo?",
+    "recover_password": "Obnoviť heslo",
+    "send_recovery_email": "Odoslať e-mail pre obnovenie",
+    "sending_recovery_email": "Odosielanie e-mailu pre obnovenie",
+    "never_mind": "Nevadí",
```

Comment:
I think "never mind" doesn't translate that well into Slovak (or Czech), so, perhaps, I would be a bit more specific here to reduce any potential confusion what the link might do 😅 

What do you think of using `Naspäť` instead, as in providing a literal "Back" link?

Reply:
@iKristy And what is the link actually doing?

Reply:
@michalhudecek: After clicking it, it leads you back to the previous screen.